development/metamaskbot-build-announce.ts

```diff
@@ -44,20 +44,19 @@ function getPercentageChange(from: number, to: number): number {
  * @returns True if the artifact exists, false if it doesn't
  */
 async function artifactExists(url: string): Promise<boolean> {
-  // Using a regular GET request here rather than HEAD because for some reason CircleCI always
```

Comment:
Nit: Perhaps we could make this a HEAD request to speed it up a little, now that we're no longer affected by the issue described in this comment